src/style.css

```diff
@@ -1,3 +1,91 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+
+  --blue-color: #00acdc;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto-Regular.ttf);
+}
+
+.navigation {
+  display: flex;
+  width: 100vw;
+  justify-content: flex-end;
+  flex-direction: row;
+  position: relative;
+  margin-right: 30px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-transform: uppercase;
+  background-color: white;
+  font-weight: bold;
+  font-size: 12px;
+  min-width: 768px;
```

Comment:
Don't fix container size (if there is no such a requirement). Here, you've set a minimum width on `.header`. This can cause issues on smaller screens. It's better to let the content dictate the size of its container.